confChamber.py

```diff
@@ -3,19 +3,23 @@
 Script to configure the VFATs on a GEM chamber
 By: Cameron Bravo c.bravo@cern.ch
 Modified by: Eklavya Sarkar eklavya.sarkar@cern.ch
+             Brian Dorney brian.l.dorney@cern.ch
 """
 
 from array import array
 from gempython.tools.vfat_user_functions_uhal import *
 from mapping.chamberInfo import chamber_vfatDACSettings
 from qcoptions import parser
+from qcutilities import readBackCheck 
 
 parser.add_option("--chConfig", type="string", dest="chConfig", default=None,
                   help="Specify file containing channel settings from anaUltraSCurve", metavar="chConfig")
 parser.add_option("--filename", type="string", dest="filename", default=None,
                   help="Specify file containing settings information", metavar="filename")
 parser.add_option("--run", action="store_true", dest="run",
                   help="Set VFATs to run mode", metavar="run")
+parser.add_option("--verify", action="store_true", dest="verify",
+                  help="Performs a readback of registers and compares stored values with written values", metavar="verify")
```

Comment:
Can the `verify` option be used standalone?  I.e., one doesn't write the config, but just checks that what are in the chips currently conforms to the expected config?
Probably a little bit of work, or maybe as simple as adding a `write`/`set` option which if true, puts the config into the chips..., but this will probably play into the comment about migrating functionality longer term into `cmsgemos/gempython`

Reply:
Yes I can change the`verify` option to be standalone.  No problem here.

I can also work on adding a `write/set` option for this PR as well.

Reply:
After thinking about this some more this would require a larger overhaul.  What is it that we want `--verify` to do?  The original idea was that if `--chConfig` and/or `--vfatConfig` was supplied it would cross-check against the input file that reg values that are read match reg values that were asked to be written.

So to have `--verify` as standalone doesn't seem to make sense since it requires an input file in each case.  Maybe you could elaborate more what you'd like it to do?

In the meantime I'll try to come up with a `write/set` option.

Reply:
Yeah, my thought was you could tell it what configuration *should* be loaded into the chips, and check that it in fact is.  It would take the same arguments as the normal `conf/write/set` process, and could be passed in along with the `conf/write/set` option to verify that the newly written config matches expectation of what was just written as well.
I think this answers my question in the review, so no need to reply to that, just fix the one change and it should be good to go.